doc/_static/css/custom.css

```diff
@@ -95,21 +114,31 @@ html[data-theme="dark"] .wy-nav-top a {
     content: none;
 }
 
-/* Swap the ordering of default RTD logo/title elements and position accordingly. */
+/* Redesign the top-left logo area. */
 .wy-side-nav-search .icon-home {
     display: flex;
-    flex-direction: row-reverse;
+    flex-direction: column-reverse;
     align-items: center;
-    justify-content: left;
 
     padding: 16px 12px;
+    padding-bottom: 32px;
 
     color: var(--color-emphasis) !important;
+
+    border-radius: 16px;
 }
 .wy-side-nav-search .icon-home img.logo {
     margin: 0 !important;
-    padding: 0px;
-    padding-right: 10px;
+    padding: 0;
+    max-width: 100px;
+    padding-bottom: 16px;
+}
+.wy-side-nav-search > div.version {
+    font-size: 85%;
+    /* Same monospace as our `pre` blocks */
+    font-family: SFMono-Regular,Menlo,Monaco,Consolas,Liberation Mono,Courier New,Courier,monospace;
```

Comment:
If it's the same, can we CSS variable it? [This post suggests](https://stackoverflow.com/questions/53846582/can-i-use-css-variables-in-a-font-list-and-have-it-work-in-legacy-browsers) it was an issue for legacy browsers, but 6 years ago.

Reply:
Kinda same answer here as above: the "same" in this comment block just refers to the general styling rules I found on the page itself via inspection; no idea which system/theme/framework sets those fonts. Thus making a variable out of this wouldn't actually affect any other part of our custom.css than this single line, so on a glance the idea doesn't really make sense to me. (although sure it can also be a small stylistic choice)

Also this is sort of a rare case in styling anyway. Normally the monospace-ness of an element should already be dictated semantically by the element itself (`pre` `code` etc.).